### DISCUSSION TITLE
Address entity extraction not doing smart formatting

### DISCUSSION THREAD

**Discussion by ritz11** [deepgram/community#331] [General help]
I am using the following Deepgram API in python:
`response = await deepgram.transcription.prerecorded(source, {'punctuate': True, 'detect_entities': True, 'tier': 'nova', 'model': 'phonecall'})`
The transcripts on inputs, I am getting are as follows:

> One fifty Main Street in Barring, Jewel As.
> Nine four nine four Humble West Road, apartment twenty six twenty one.

The entities returned do not give the full address as entity and rather parts. Is there a way or some configuration I can get the transcription formatted?

**Comment by jkroll-deepgram** [comment]
@ritz11, you're using the correct feature, as our [entity detection](https://developers.deepgram.com/docs/detect-entities) does support addresses.

Can you describe or share how you're seeing the entities within the transcript? I'd expect to see them in `response["results"]["channels"][0]["alternatives"][0]["entities"]`, with `"label": "LOCATION_ADDRESS"`. Are the examples you're sharing from the transcript's `words`, or from the transcript's `entities`?
